src/PeopleSDKAdapter.js

```diff
@@ -0,0 +1,119 @@
+import {Observable, from, fromEvent, merge} from 'rxjs';
+import {PeopleAdapter, PersonStatus} from '@webex/component-adapter-interfaces';
+import {publish, refCount, flatMap, map, filter} from 'rxjs/operators';
+
+const USER_PRESENCE_UPDATE_EVENT = 'event:apheleia.subscription_update';
+
+/**
+ * The `PeopleSDKAdapter` is an implementation of the `PeopleAdapter` interface.
+ * This adapter utilizes the Webex JS SDK to fetch data about a person.
+ *
+ * @export
+ * @class PeopleSDKAdapter
+ * @extends {PeopleAdapter}
+ */
+export default class PeopleSDKAdapter extends PeopleAdapter {
+  constructor(datasource) {
+    super(datasource);
+
+    this.getPersonObservables = {};
+  }
+
+  /**
+   * Returns a proper Enum key for a given PersonStatus value
+   *
+   * @param {string} status
+   * @returns {string} PersonStatus
+   * @memberof PeopleSDKAdapter
+   */
+  getStatus(status) {
+    return Object.keys(PersonStatus).find((key) => PersonStatus[key] === status);
+  }
+
+  /**
+   * Encodes the personID into base64 format and return the UUID
```

Comment:
If I'm reading the code correctly, this is actually decoding the base64 string then grabbing the last section. Probably would help to have some samples in the jsdoc.

Reply:
I think it's documenting strings should keep the implementation details out of the description (as you can get that by reading the code), and instead focus on the result.
```suggestion
   * Returns the UUID of a person from the given public ID
```
Not super sure on the subtleties of UUID vs hydra ID but i hope you get the idea.

As for the examples, if we are going to do that, let's do it only on the "public" ones